caikit/runtime/train.py

```diff
@@ -44,10 +45,34 @@
 log = alog.use_channel("TRAIN")
 error = error_handler.get(log)
 
+USER_ERROR_EXIT_CODE = 1
+INTERNAL_ERROR_EXIT_CODE = 200
```

Comment:
NIT: Was there a reason you chose 200? Seems like that might confuse some folks w.r.t. HTTP 200

Reply:
This is true. It just needed to be a number >= 128. (generally considered a system/retryable error by projects like Kubeflow Training Operator). I can see the confusion around 200 = HTTP OK! I can change it to something else

Reply:
Ah, makes sense. It would be good to leave a note about that in the code!